daemon/bpf.sha

```diff
@@ -1,2 +1,2 @@
-GO_BINDATA_SHA1SUM=9531ba86f751cad1eb25fe2b913f13a448aab4e6
-GO_VERSION_USED=go1.9
+GO_BINDATA_SHA1SUM=4846bd2f17b1bb2c3f3722a631169917b2c29ba3
+GO_VERSION_USED=go1.8.3
```

Comment:
Will this go version `1.8.3` affect other developers? Is this `GO_VERSION_USED` being used one a location that I'm not aware of?

Reply:
I'll update to go1.9. It's not used anywhere, it's only used for us.